gradle-plugin/src/main/kotlin/io/reflekt/plugin/ReflektSubPlugin.kt

```diff
@@ -47,24 +57,24 @@ class ReflektSubPlugin :  KotlinCompilerPluginSupportPlugin {
     override fun getPluginArtifact(): SubpluginArtifact = SubpluginArtifact(
         groupId = GRADLE_GROUP_ID,
         artifactId = GRADLE_ARTIFACT_ID,
-        version = VERSION
+        version = VERSION,
     )
 
     private fun getFilesToIntrospect(jarFiles: Set<File>): List<File> {
         val files: MutableList<File> = ArrayList()
         jarFiles.forEach { jar ->
             getSourceJar(jar)?.let {
-                files.addAll(extractAllFiles(it))
+                files.addAll(it.extractAllFiles())
             }
         }
         return files
     }
 
-    private fun getSourceJar(jarFile: File) : File? {
+    private fun getSourceJar(jarFile: File): File? {
         val sourceName = "${jarFile.name.substringBeforeLast('.', "")}-sources.jar"
-        jarFile.parentFile.parentFile.listFiles()?.filter{ it.isDirectory }?.forEach { folder ->
+        jarFile.parentFile.parentFile.listFiles()?.filter { it.isDirectory }?.forEach { folder ->
             val sources = folder.listFiles()?.find { it.name == sourceName }
-            if (sources != null) {
```

Comment:
Please also keep an eye on this inspection. As Kotlin has special language specific null-safe logic, all  Java-like null comparissons are converted into corresponding (`let`/`run`/etc.). If you don't like this one, I will supress it also,but actually I suppose that these features should be used as you are writting Kotlin code, but not Java code.